packages/vue-cli-plugin-typescript/generator/template/src/EngineService.ts

```diff
@@ -1,47 +1,25 @@
+/* eslint-disable @typescript-eslint/explicit-module-boundary-types */
```

Comment:
Why, precisely?
It _can_ be the solution for 'Removed all the missing return type warnings', I get that, but that's the last resort solution.
To approves that I need to know exactly what prevents to just comply with this rule.

Reply:
I also don't like this approach as well, but I made sure to minimize the amount of eslint-disable rules.
In this case, the `getEngine` function has the following return type: `HeadlessEngine<ReducersMapObject<SearchAppState, Action<any>>>`. 
However, `ReducersMapObject` is not exposed in Coveo Headless, hence the `ReturnType<typeof getEngine>`.